### PR TITLE
Fix ApiKeys page issues from bug bash

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes the following issues:
- #5170: JS exception from ApiKeys page (edit)
- #5320: pointless nested container
- #5322: horizontal rule is ugly
- #5313: partial validation
- #5315: Edit doesn't show org packages

Screenshot of no nested container, partial validation gone:
image

Screenshot of removed bottom border at end:
image

### CODE REVIEW

File: src/NuGetGallery/Scripts/gallery/page-api-keys.js [hunk 1]
```diff
@@ -92,7 +92,19 @@
...
+                this._SetPackageSelection(this.Packages());
+
+                if (this.Owner()) {
+                    var existingOwner = ko.utils.arrayFirst(
+                        this.PackageOwners,
+                        function (owner) {
+                            return owner.Owner.toUpperCase() == data.Owner.toUpperCase()
+                        });
+                    this.PackageOwner(existingOwner);
+                }
```
Comment (scottbommarito) [inline src/NuGetGallery/Scripts/gallery/page-api-keys.js]: nit: `} else {`

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 2]
```diff
@@ -120,7 +118,7 @@
 </script>
 
 <script type="text/html" id="api-key-details">
-    <div class="api-key-details">
+    <div class="api-key-details" data-bind="css: { 'api-key-details-append': $index() > 0 }">
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: There are selectors that allow you to do this in CSS without having to touch the knockout. Take a look at `page-statistics-most-downloaded.less` and https://stackoverflow.com/questions/5065766/css3-selector-question-for-all-but-first-select.

If reformatting this to use the selectors is too difficult, this is ok as is.

File: src/NuGetGallery/Views/Users/ApiKeys.cshtml [hunk 3]
```diff
@@ -44,10 +44,8 @@
                     "create",
                     "Create",
                     @<text>
-                        <div class="panel panel-default upsert-api-key">
-                            <div class="panel-body"
-                                 data-bind="template: { name: 'upsert-api-key', data: NewApiKey }">
-                            </div>
+                        <div class="upsert-api-key">
```
Comment (shishirx34) [inline src/NuGetGallery/Views/Users/ApiKeys.cshtml]: unrelated to PR, what is an 'upsert'? I see that class all over the place.

Comment (chenriksson) [reply src/NuGetGallery/Views/Users/ApiKeys.cshtml]: Apparently folks much younger than I invented the [word](https://www.urbandictionary.com/define.php?term=upsert) to mean update or insert. :-)

**Pull request merged**